script_facade/models/r1/medication_order.py

```diff
@@ -8,56 +8,75 @@
 }
 
 class MedicationOrder(object):
+    def __init__(self):
+        # required attribute
+        # https://www.hl7.org/fhir/DSTU2/medicationorder-definitions.html#MedicationOrder.medication_x_
+        self.medication = None
 
-    def __init__(self, med, dispense_request=None, prescriber=None, date_written=None, date_ended=None):
-        # todo: support medicationReference and medicationCodeableConcept
-        self.medication = med
-        self.dispense_request = dispense_request
-        self.prescriber = prescriber
-
-        self.date_written = date_written
+        self.date_written = None
```

Comment:
I'm sure you mean to take the values given.  i.e. `self.date_written = date_written` (which will be None if the caller doesn't include a better `date_written` param).

Reply:
I'll remove this line, and the lines the follow, that set [MedicationOrder](https://cosri-pdmp.cirg.washington.edu/v/r2/fhir/MedicationOrder) attributes (`date_written`, `date_ended`, `dispense_request`, `prescriber`) that aren't required by FHIR.

...or is it better to initialize these all to `None`, so I don't have to check if each instance has them set?...

Reply:
after discussion, setting all attributes to `None`